src/pages/kwonojae/Login/Login.scss

```
@@ -0,0 +1,60 @@
+@import '../../../styles/Variable.scss';
+
+.loginOJ {
+  @extend .flexCS;
+  align-items: center;
+  width: 350px;
+  height: 380px;
+  margin: 150px auto 0px auto;
+  border: $border1;
+  border-radius: 1px;
+  h1 {
+    margin: 32px 0px 0px 0px;
```

```suggestion
    margin: 32px 0 0 0
```
속성값 적으실 때 0은 굳이 px 표시 안해주셔도 됩니다!

그리고 short hand 사용하시면 좌우나 상하 같은값이 존재할 경우에는 따로 안적어 주셔도 됩니다! 
( 순서: 위 오른 아래 왼)

---

👍

---

수정 했습니다~ 자세한 설명 감사합니다 ㅎ